src/components/pages/Question.jsx

```diff
@@ -0,0 +1,41 @@
+
+export default function Question(props) {
+  
+  return (
+    <div className="border-b-2 border-gray-300 pb-4 max-w-7xl">
+      <p className="text-lg">{props.element.question}</p>
+      <div className="flex gap-x-6 pt-4">
+        {props.element.randomAnswers.map((element, index) => {
+          const selected =props.element.chosenAnswer === element;
+          const correctanswer = element === props.element["correct_answer"];
+          
```

Comment:
By using destructuring, you can avoid repeating the long `props.element` object multiple times in your code. Additionally, you can rename the `correct_answer `key to `correctAnswer` to follow standard JavaScript naming conventions. This will make your code more readable and easier to understand. 
Check this suggestion on how you can improve this: 

```suggestion
       const { chosenAnswer, correct_answer: correctAnswer } = props.element;
       const selected = chosenAnswer === element;
       const isCorrect = element === correctAnswer;
```